### DISCUSSION TITLE
Rails guides auto updating ebook

### DISCUSSION THREAD

**Discussion by rctneil** [rails/rails#53932] [Ideas]
Hi,

I know the rails guides can be downloaded as an epub, but is there a version that auto updates when the guides get updated?

Eg, I mean like how books from the iBook store get updated.

Thanks

**Comment by fatkodima** [comment]
There is no.
Please use stackoverflow or http://discuss.rubyonrails.org for such questions. Rails "issues" is reserved for bugs within the framework.

**Comment by rctneil** [comment]
Ok,   understood.   Could there be one?    Sounds like it would be super useful to a lot of people.   Bit of a shame to just dismiss the idea so fast when it could be of help to people if done?

**Comment by zzak** [comment]
@rctneil We're not trying to dismiss your idea, but issues are not the place to ask for feedback on ideas. If you want a self-updating version of the guides ebook it would have to be hosted somewhere and need a way to deploy that when changes are merged. Given the amount of effort there, I don't think we're interested in, as there are efforts focused on to update guides/edgeguides currently planned and ongoing.
